src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/centos/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -0,0 +1,41 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  
+  <PropertyGroup>
+    <Version>1.0.1</Version>
+    <RuntimeOSPath>$(OutputRootPath)CentOs.7.1.1503</RuntimeOSPath>
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
+    <PackageTargetRuntime>centos.7.1-$(PackagePlatform)</PackageTargetRuntime>
```

Comment:
When you change this stuff to rhel, please use `rhel.7` as the prefix in the RID.  See dotnet/corefx@f71d339aa587c9d6c1ea12441bada92443a26db9 for the changes I made for RHEL support in CoreFX.